blueflood-core/src/main/java/com/rackspacecloud/blueflood/outputs/handlers/RollupHandler.java

```diff
@@ -129,6 +143,8 @@ private rollupsOnReadTimers (Timer timer) {
         List<SearchResult> units = null;
         List<Locator> locators = new ArrayList<Locator>();
 
+        Timer.Context c = timerRorCalcUnits.time();
```

Comment:
I added this to ascertain the exact time spent accessing elastic search as it can take up to 100-200 ms in some cases.

I felt more comfortable being able to reference an exact metric rather than calculating the differences of other ones.